api/app/Services/ModelFactoryInstance.php

```diff
@@ -248,6 +248,12 @@ public function transformed()
             }
 
             $this->setEntityType();
+        } elseif ($this->factoryInstance instanceof BaseModel) {
+            $entity = $this->factoryInstance;
```

Comment:
Yea it probably will break the documentation for the timezones, so communicate with @artstorm on that. From memory it was implemented so the docs could use the actual values returned from the data service, but there is no reason that they shouldn't have a mock value in the docs.